pkg/reconciler/taskrun/resources/taskref.go

```diff
@@ -217,9 +217,9 @@ type LocalTaskRefResolver struct {
 
 // GetTask will resolve either a Task or ClusterTask from the local cluster using a versioned Tekton client. It will
 // return an error if it can't find an appropriate Task for any reason.
-// TODO: if we want to set source for in-cluster task, set it here.
+// TODO: if we want to set RefSource for in-cluster task, set it here.
```

Comment:
This issue is closed, do we still need it?

Reply:
Thanks @Yongxuanzhang ! I've removed it.